### PR TITLE
[MAINT] Packaging

### PR DESCRIPTION
**Pull request by emdupre**

### CODE REVIEW

File: fmralign/_utils.py [hunk 1]
```diff
@@ -178,49 +198,64 @@ def _make_parcellation(imgs, clustering_index, clustering, n_pieces, masker, smo
...
+                n_parcels=n_pieces,
+                mask=masker,
+                scaling=False,
+                n_iter=20,
+                verbose=verbose,
+            )
         except TypeError:
-            if clustering == "rena" and (version.parse(nilearn.__version__) <= version.parse("0.5.2")):
+            if clustering == "rena" and (
+                version.parse(nilearn.__version__) <= version.parse("0.5.2")
```
Comment (bthirion) [inline fmralign/_utils.py:230]: We should not consider such old nilearn versions.

**Review by bthirion — COMMENTED** [review]
LGTM besides 2 comments.
Thx for the cleaning !

**Comment by emdupre** [comment]
Thanks !! I just pushed a commit to remove references to nistats and older nilearn versions.

I'm just going to add a pre-commit for black and isort and then I think this is ready to go. I'll update the docs in a separate PR.

**Comment by emdupre** [comment]
OK, I'm going to merge this before it gets any bigger !

Confirming that I can `pip install -e .` with this repo and import successfully. Now to get the CI running and docs building....

**Pull request merged**